tests/saitests/py3/sai_base_test.py

```diff
@@ -32,6 +32,17 @@
 from switch import (sai_thrift_port_tx_enable,      # noqa E402
                     sai_thrift_port_tx_disable)
 
+DATA_PLANE_QUEUE_LIST = ["0", "1", "2", "3", "4", "5", "6", "7"]
+
```

Comment:
Will the LAG go down if q 7 is disabled?

Reply:
No.

Reply:
Thanks for checking. Can you elaborate on that?

Reply:
In short, this affects data plane only.